packages/web/src/components/organisms/PropertySelectForm/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import Link from 'next/link'
+import { Button } from 'antd'
+import { AuthorSelector } from './AuthorSelector'
+import { PropertyCreateForm } from './PropertyCreateForm'
+import { getAccountAddress } from 'src/fixtures/wallet/utility'
+import { useEffectAsync } from 'src/fixtures/utility'
+
+interface Props {
+  market: string
+}
+
+export const PropertySelectForm = ({ market }: Props) => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [author, setAuthor] = useState<string>('')
+  const [property, setProperty] = useState<string>()
+  const [query, setQuery] = useState<string>('')
+  const [isCreated, setIsCreated] = useState(false)
+  const onChange = (value: string) => setProperty(value)
+  const onSearch = (query: string) => setQuery(query)
+  const onSubmit = (createdPropertyAddress: string) => {
+    setProperty(createdPropertyAddress)
+    setIsCreated(true)
+  }
+  useEffectAsync(async () => {
+    const accountAddress = await getAccountAddress()
+    const dafaultValue = process.env.NODE_ENV === 'development' ? '0x69Cc2C86aeB26f52F6645a2DFdec1051DD5584C0' : ''
```

Comment:
I couldn't get account address,  so defined `0x69Cc2C86aeB26f52F6645a2DFdec1051DD5584C0` as default value. Could you please check to see if you can get this value in your environment?